.changeset/afraid-gifts-reply.md

```diff
@@ -0,0 +1,5 @@
+---
+"@wso2is/admin.tenants.v1": patch
```

Comment:
Lets add a Console changeset as well

Reply:
Fixed in https://github.com/wso2/identity-apps/pull/8079/commits/6210f70a92166a192b748a35541b26ed26c9c039